demo/cms/components/primitives.jsx

```diff
@@ -95,6 +96,11 @@ const RelationshipAnnotator = ({ type, value }) => (
 primitives.Relationship.setRelationshipAnnotator(RelationshipAnnotator);
 
 module.exports = {
+	SystemLifecycle: {
```

Comment:
```suggestion
	AnEnum: {
```
The keys of this object need to be the names of actual primitive types or enums.

**Warning!** may end up breaking some existing tests so might be more straightforward to define a new type of enum in enums.yaml and add another property to MainType.yaml that uses this new enum